spec/request/users_controller_spec.rb

```diff
@@ -39,17 +39,34 @@
     end
 
     it "should create a user and redirect" do
+      akismet_stub_response_ham
       post "/users", params: params
 
       expect(response.status).to eq(302)
       expect(response).to redirect_to(login_url(already_joined: true))
     end
 
     it "should raise an error if user is invalid" do
+      akismet_stub_response_ham
       post "/users", params: { user: { login: 'bar', password: 'foo' } }
 
       expect(flash[:error]).to be_present
       expect(response).to render_template("users/new")
     end
+
+    it "should raise an error if user fails Akismet check" do
+      akismet_stub_response_spam
+      post "/users", params: { user: { login: 'bar', password: 'foo' } }
+
+      expect(flash[:error]).to be_present
```

Comment:
`flash[:error]` looks to be present here for another reason like missing form fields (we aren't yet explicitly adding to the flashes when a user is marked as spam)

Reply:
+1